app/src/main/java/com/eveningoutpost/dexdrip/stats/StatsResult.java

```diff
@@ -75,14 +77,22 @@ public StatsResult(SharedPreferences settings, long from, long to){
             avg = 0;
         }
 
+        if(getTotalReadings() > 0){
+            cursor= db.rawQuery("select ((count(*)*(sum(calculated_value * calculated_value)) - (sum(calculated_value)*sum(calculated_value)) )/((count(*)-1)*(count(*))) ) from bgreadings  where timestamp >= " + from + " AND timestamp <= " + to + " AND calculated_value > " + DBSearchUtil.CUTOFF + " AND snyced == 0", null);
+            cursor.moveToFirst();
+            stdev = cursor.getDouble(0);
+            stdev = Math.sqrt(stdev);
```

Comment:
Yes, as @jamorham suggested, I'll move it into its own method so it is only calculated when required.  I added it for the History pages.  And actually, I would like to see it on the Home screen as well, so I've enabled it via a preference.